src/components/Booking.js

```diff
@@ -56,10 +57,10 @@ class Booking extends React.Component {
     axios.defaults.xsrfCookieName = 'csrftoken';
     axios.defaults.headers = {
       'Content-Type': 'application/json',
-      Authorization: `Token ${this.props.token}`,
+      Authorization: `Bearer ${this.props.authToken}`,
```

Comment:
Found the same problem in the Booking file as well, so fixed that and also the hardcoded localhost API request address at the same time.